.github/workflows/AtlasBuildTest.yml

```diff
@@ -0,0 +1,25 @@
+name: Testing Atlas Build
+
+on:
+    push:
+        paths:
+            - "packages/theming/atlas/**"
```

Comment:
I would only trigger this workflow for PR's and push to master branch:
```
on:
    push:
        branches: [ master ]
    pull_request:
        branches: [ master ]
```
This also happens in other workflows.

Reply:
I implemented that way because the idea/suggestion was only calls this workflow when we have some changes inside the atlas folder.